### DISCUSSION TITLE
Welcome to Vulkan-Samples Discussions!

### DISCUSSION THREAD

**Discussion by ghost** [KhronosGroup/Vulkan-Samples#232] [General]
## 👋 Welcome to the Vulkan Samples!
  We’re using Discussions as a place to discuss issues, PRs and any feature or sample requests. We hope that you:
  * Ask questions about the samples
  * Discuss issues you are having
  * Raise possible new features or samples
  * Engage with other community members 
  * Follow the [Khronos Code of Conduct](https://www.khronos.org/events/code-of-conduct)

**Comment by SB2020-eye** [comment]
Hi. I don't know if this is the correct place to ask this or not. Consider me a noob on many fronts.

I am trying to run a github repo, https://github.com/nihui/realsr-ncnn-vulkan. I _would_ be trying to run https://github.com/jixiaozhong/RealSR or https://github.com/Tencent/Real-SR, but these require GPU + CUDA, and I only have an integrated graphics card. Anyways, even the 1st repo I referenced gave me an error -- something like "vulkan-1.dll not found".

I have a Lenovo Yoga 3 Pro that has an Intel Core M-5Y71 processor and Intel HD Graphics 5300.

I believe I'm seeing from Intel and the like that Vulkan isn't supported. But I'm not 100% sure. And even if this is the case _in general_, I'm open to trying workarounds and the like.

Again, if this is the wrong place for this subject, my apologies. I'll even delete it if needed (and if possible). In any case, any direction would be much appreciated.

All the best.

**Reply by SaschaWillems** [reply]
This might not be the best place for such a question. But afaik the Core M-5Y71 is a Broadwell CPU which does not support Vulkan on Windows at all. There is very basic support for it on a few [Linux distrubitons](https://vulkan.gpuinfo.org/listreports.php?devicename=Intel%28R%29+HD+Graphics+5300+%28Broadwell+GT2%29), but in general you probably don't want to run Vulkan on such an old iGPU at all.

**Comment by flo-th** [comment]
Hi all,

I built the apk file of the vulkan samples, installed it on my device (remote access without x11 forwarding) and i am wondering which adb command to perform to start a simple test which could ensure me that my device is well able to create a vulkan device and then validate the gpu rendering/acceleration ?
I tried to run Swapchain Images sample on an Android device but i have no returned resulting of this call.

Thanks.
